aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -1148,11 +1171,13 @@ Tensor argmax(const Tensor& self, c10::optional<int64_t> dim, bool keepdims) {
 }
 
 Tensor& argmin_out(const Tensor& self, c10::optional<int64_t> dim, bool keepdim, Tensor& result) {
-  TORCH_CHECK(self.numel() > 0, "cannot perform reduction function argmin on a "
-      "tensor with no elements because the operation does not have an identity");
   Tensor in;
   if (dim) {
     auto sizes = self.sizes();
+    if (self.ndimension() != 0) {
```

Comment:
My comment doesn't seem to have been addressed https://github.com/pytorch/pytorch/pull/52565#discussion_r594405175

Reply:
Ah my bad. Missed your reply.

Reply:
Fixed in latest commit.